package.json

```diff
@@ -38,7 +38,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "express": "4.17.1",
+    "express": "4.17.2",
     "lodash": "4.17.21",
     "redis": "2.8.0"
```

Comment:
Could you please update Redis to v3 (at less).

Reply:
I'll try again, but the somehow the API does not really like it

Reply:
API, is already in v3 :)